--- CMakeLists.txt
+++ CMakeLists.txt
@@ -8,6 +8,8 @@
 
 cmake_minimum_required(VERSION 3.1)
 
+set(CMAKE_MODULE_PATH ${CMAKE_BINARY_DIR} ${CMAKE_MODULE_PATH})
+
 # define the project
 project(
         PagedGeometry
@@ -100,7 +102,7 @@
 ENDIF ()
 
 # --- Ogre 3D graphics engine ---
-find_package(OGRE REQUIRED CONFIG)
+find_package(OGRE REQUIRED)
 
 # now add the directories
 add_subdirectory(source)
