--- CMakeLists.txt
+++ CMakeLists.txt
@@ -7,7 +7,7 @@
 # cmake system for PagedGeometry updated on 6-10-2017 by Edgar{at}AnotherFoxGuy{DOT}com
 
 cmake_minimum_required( VERSION 3.0.2 )
-set( CMAKE_MODULE_PATH ${CMAKE_MODULE_PATH} ${CMAKE_SOURCE_DIR}/cmake )
+set( CMAKE_MODULE_PATH ${CMAKE_MODULE_PATH} ${CMAKE_BINARY_DIR} ${CMAKE_SOURCE_DIR}/cmake )
 
 
 # add some functions we use that are shipped with cmake
@@ -20,7 +20,7 @@
 # some versioning things
 SET(LIB_MAJOR_VERSION "1")
 SET(LIB_MINOR_VERSION "2")
-SET(LIB_BUILD_VERSION "0")
+SET(LIB_BUILD_VERSION "1")
 SET(LIB_VERSION "${LIB_MAJOR_VERSION}.${LIB_MINOR_VERSION}.${LIB_BUILD_VERSION}")
 IF(NOT DEFINED LIB_INSTALL_DIR)
   SET(LIB_INSTALL_DIR "lib${LIB_SUFFIX}")
