--- MyGUIEngine/CMakeLists.txt
+++ MyGUIEngine/CMakeLists.txt
@@ -1,5 +1,19 @@
 set (PROJECTNAME MyGUIEngine)
 
+# FreeType from conan has seperate libraries
+find_package(PNG)
+find_package(BZip2)
+
+find_library(BROTLICOMMON_LIBRARY NAMES brotlicommon brotlicommon-static)
+find_library(BROTLIDEC_LIBRARY NAMES brotlidec brotlidec-static)
+
+if(NOT ${BROTLIDEC_LIBRARY} STREQUAL "BROTLICOMMON_LIBRARY-NOTFOUND" AND NOT ${BROTLIDEC_LIBRARY} STREQUAL "BROTLIDEC_LIBRARY-NOTFOUND")
+	set(BROTLI_LIBRARIES ${BROTLICOMMON_LIBRARY} ${BROTLIDEC_LIBRARY})
+	message(STATUS "Brotli libraries found: ${BROTLI_LIBRARIES}")
+else()
+	message(WARNING "Brotli library not found: ${BROTLICOMMON_LIBRARY} ${BROTLIDEC_LIBRARY}")
+endif()
+
 project( ${PROJECTNAME} )
 
 include(${PROJECTNAME}.list)
@@ -42,10 +56,12 @@
 		target_link_libraries(${PROJECTNAME} msdfgen)
 	endif ()
 
-	target_link_libraries(${PROJECTNAME} ${FREETYPE_LIBRARIES})
-	if (ZLIB_FOUND) # hacky way to check if freetype was built with zlib
-		target_link_libraries(${PROJECTNAME} ${ZLIB_LIBRARIES})
-	endif()
+	target_link_libraries(${PROJECTNAME} 
+			${PNG_LIBRARIES}
+			${BZIP2_LIBRARIES} 
+			${BROTLI_LIBRARIES} 
+			${FREETYPE_LIBRARIES})
+	
 endif()
 
 # platform specific dependencies
