--- Components/Overlay/CMakeLists.txt
+++ Components/Overlay/CMakeLists.txt
@@ -7,6 +7,21 @@
 # free to make use of it in any way you like.
 #-------------------------------------------------------------------
 
+
+# FreeType from conan has seperate libraries
+find_package(PNG)
+find_package(BZip2)
+
+find_library(BROTLICOMMON_LIBRARY NAMES brotlicommon brotlicommon-static)
+find_library(BROTLIDEC_LIBRARY NAMES brotlidec brotlidec-static)
+
+if(NOT ${BROTLIDEC_LIBRARY} STREQUAL "BROTLICOMMON_LIBRARY-NOTFOUND" AND NOT ${BROTLIDEC_LIBRARY} STREQUAL "BROTLIDEC_LIBRARY-NOTFOUND")
+  set(BROTLI_LIBRARIES ${BROTLICOMMON_LIBRARY} ${BROTLIDEC_LIBRARY})
+  message(STATUS "Brotli libraries found: ${BROTLI_LIBRARIES}")
+else()
+  message(WARNING "Brotli library not found: ${BROTLICOMMON_LIBRARY} ${BROTLIDEC_LIBRARY}")
+endif()
+
 ############################################################
 # Overlay optional component
 ############################################################
@@ -52,7 +67,7 @@
 
 if(FREETYPE_FOUND)
   target_compile_definitions(OgreOverlay PRIVATE HAVE_FREETYPE)
-  target_link_libraries(OgreOverlay PRIVATE ${FREETYPE_LIBRARIES})
+  target_link_libraries(OgreOverlay PRIVATE ${FREETYPE_LIBRARIES} ${PNG_LIBRARIES} ${BZIP2_LIBRARIES} ${BROTLI_LIBRARIES} ZLIB::ZLIB)
   target_include_directories(OgreOverlay PRIVATE ${FREETYPE_INCLUDE_DIRS})
 elseif(UNIX)
   set_source_files_properties(src/OgreFont.cpp PROPERTIES
