--- CMake/Dependencies.cmake
+++ CMake/Dependencies.cmake
@@ -13,7 +13,7 @@
 
 # OGRE_DEPENDENCIES_DIR can be used to specify a single base
 # folder where the required dependencies may be found.
-set(OGRE_DEPENDENCIES_DIR "${PROJECT_BINARY_DIR}/Dependencies" CACHE PATH "Path to prebuilt OGRE dependencies")
+set(OGRE_DEPENDENCIES_DIR ${CMAKE_PREFIX_PATH})
 option(OGRE_BUILD_DEPENDENCIES "automatically build Ogre Dependencies (SDL2, pugixml)" TRUE)
 
 message(STATUS "DEPENDENCIES_DIR: ${OGRE_DEPENDENCIES_DIR}")
@@ -68,7 +68,7 @@
 set(CMAKE_PREFIX_PATH ${CMAKE_PREFIX_PATH} ${OGRE_DEP_SEARCH_PATH})
 set(CMAKE_FRAMEWORK_PATH ${CMAKE_FRAMEWORK_PATH} ${OGRE_DEP_SEARCH_PATH})
 
-if(OGRE_BUILD_DEPENDENCIES AND NOT EXISTS ${OGREDEPS_PATH})
+if(0)
     message(STATUS "Building pugixml")
     file(DOWNLOAD
         https://github.com/zeux/pugixml/releases/download/v1.10/pugixml-1.10.tar.gz