--- CMakeLists.txt      
+++ CMakeLists.txt
@@ -16,6 +16,8 @@
 
 cmake_minimum_required(VERSION 3.3.0)
 
+include(${CMAKE_BINARY_DIR}/conan_paths.cmake)
+
 set(CMAKE_CXX_STANDARD 11)
 set(CMAKE_CXX_STANDARD_REQUIRED ON)
 set(CMAKE_CXX_EXTENSIONS OFF)
@@ -63,6 +65,7 @@
 
 # Include necessary submodules
 set(CMAKE_MODULE_PATH
+  "${CMAKE_BINARY_DIR}"
   "${PROJECT_SOURCE_DIR}/CMake"
   "${PROJECT_SOURCE_DIR}/CMake/Utils"
   "${PROJECT_SOURCE_DIR}/CMake/Packages"

