--- CMakeLists.txt
+++ CMakeLists.txt
@@ -7,7 +7,7 @@
 # cmake system for PagedGeometry updated on 2-9-2021 by Edgar{at}AnotherFoxGuy{DOT}com
 
 cmake_minimum_required(VERSION 3.0.2)
-set(CMAKE_MODULE_PATH ${CMAKE_MODULE_PATH} ${CMAKE_SOURCE_DIR}/cmake)
+set(CMAKE_MODULE_PATH ${CMAKE_BINARY_DIR} ${CMAKE_MODULE_PATH} ${CMAKE_SOURCE_DIR}/cmake)
 
 # define the project
 project(
@@ -101,7 +101,7 @@
 ENDIF ()
 
 # --- Ogre 3D graphics engine ---
-find_package(OGRE REQUIRED CONFIG)
+find_package(OGRE REQUIRED)
 
 # now add the directories
 add_subdirectory(source)
